includes/rest-api/mappers/class-sensei-rest-api-home-controller-mapper.php

```diff
@@ -112,4 +112,55 @@ public function map_promo_banner( Sensei_Home_Promo_Banner $banner ): array {
 			'is_visible' => $banner->is_visible(),
 		];
 	}
+
+	/**
+	 * Maps a Sensei_Home_Tasks to a basic array structure to be used as response for the REST API.
+	 *
+	 * @param Sensei_Home_Tasks $tasks The tasks structure.
+	 *
+	 * @return array
+	 */
+	public function map_tasks( Sensei_Home_Tasks $tasks ): array {
+		$mapped = [];
+		foreach ( $tasks->get_items() as $task ) {
+			$mapped[ $task->get_id() ] = $this->map_task( $task );
+		}
```

Comment:
WDYT about maybe returning an array/list ordered by `priority` here, instead of mapped by ID?

Reply:
I thought about that, but that way it would be difficult for people using the filter to "remove" one, for example.
That's the main idea on having the ID, so we can uniquely identify a specific task.

Reply:
Hey there, hmmm, good point. I think we would need to remove the task only on the backend, using the filter, right?

In this case, maybe the improvements [here](https://github.com/Automattic/sensei/pull/5815#discussion_r985902964) could help to remove the need for mapping by ID first?